pkg/controller/installation/types.go

```diff
@@ -52,6 +54,7 @@ func newManagedType() *Type {
 		productOrder: [][]v1alpha1.ProductName{
 			{v1alpha1.ProductRHSSO},
 			{v1alpha1.ProductCodeReadyWorkspaces, v1alpha1.ProductFuse},
+			{v1alpha1.ProductLauncher, v1alpha1.ProductCodeReadyWorkspaces},
```

Comment:
CodeReady is included twice here, and Launcher is in a third stage. Should Launcher be included in the second stage on L56?

Reply:
These will be changed as the PR is still in progress. Launcher will be included in the second stage and the third one should be removed